pkg/ctl/utils/update_cluster_endpoint_access.go

```diff
@@ -0,0 +1,135 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/kris-nova/logger"
+	"github.com/spf13/pflag"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+)
+
+func updateEndpointAccessCmd(cmd *cmdutils.Cmd) {
+	cfg := api.NewClusterConfig()
+	cmd.ClusterConfig = cfg
+
+	cmd.SetDescription("update-cluster-api-access", "Update cluster API endpoint configuration", "")
+
+	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
+		cmdutils.AddNameFlag(fs, cfg.Metadata)
```

Comment:
this should be `--cluster` here (which will be fixed globally in #1329)

Reply:
Keeping this the same (using --name) for now so the change can happen all at once rather than piecemeal.

Reply:
Yes, let's use `--name` here, same as in `eksctl utils update-cluster-logging`. I agree that consistency can be improved going forward, but I'd rather rename once we settled on what the consistent solution should be.

Reply:
https://github.com/weaveworks/eksctl/pull/1363

Reply:
Sorry I missed this comment previously.  @rndstr, are you suggesting that we make the change in this PR or should we continue with the plan of fixing this all in #1363.  Or is this just in reply to @errordeveloper?

Reply:
> should we continue with the plan of fixing this all in #1363

this, or whichever gets merged last. if we merge #1363 first then we need to incorporate this here, otherwise there.

Reply:
I think that this is the last bit we need to merge this particular PR, with the VPC-peering and customer network <-> VPC configuration coming in the next.